GetServiceByRevision/handler.ts

```diff
@@ -136,9 +139,14 @@ export function GetServiceByRevision(
   serviceModel: ServiceModel
 ): express.RequestHandler {
   const handler = GetServiceByRevisionHandler(serviceModel);
-  const middlewaresWrap = withRequestMiddlewares(
+  const middlewaresWrap = withRequestMiddlewares<
+    "IResponseErrorValidation",
+    "IResponseErrorValidation",
+    string & INonEmptyStringTag,
+    number & INonNegativeIntegerTag
+  >(
     RequiredParamMiddleware("serviceid", NonEmptyString),
-    RequiredParamMiddleware("version", IntegerFromString)
+    RequiredParamMiddleware("version", NonNegativeIntegerFromString)
```

Comment:
Why are we introducing such change in this PR?

Reply:
The old code used the IntegerFromString io-style type to decode the input version (a string) into a number. The result is gived as input to a function parameter declared as non negative (using the INonNegativeIntegerTag type). In the strict mode, the compiler notice that IntegerFromString is not tagged properly, so I use the NonNegativeIntegerFromString type in order to properly decode the input string a non negative integer.